examples/feature_demo/aomap.py

```diff
@@ -0,0 +1,108 @@
+"""
+Ambient occlusion
+========
```

Comment:
Title underline must be the same length as the title (this makes Sphinx complain and causes the doc build to fail 😄).
```suggestion
Ambient occlusion
=================
```

Reply:
Ah, I finally know the reason.